src/main/java/com/example/demo/matcher/models/NewOrderParams.java

```diff
@@ -12,8 +11,7 @@
 
 @ToString @AllArgsConstructor @Getter
 public class NewOrderParams {
-    @UUID(message = "Bad UUID format")
-    String account;
+    String username;
```

Comment:
Are we not having any restrictions when it comes to usernames? `@Size` could be something you add here so we're still doing some sort of validation. Completely up to you though! Not a blocker just something to consider 👍

Reply:
This is something I will think about. I've made an issue already https://github.com/pmarsh-scottlogic/JavaTraining/issues/17